package.json

```diff
@@ -1,9 +1,6 @@
 {
-  "name": "@angular/dev-infra-private",
+  "name": "@angular/build-tooling",
```

Comment:
Should we create this package in its own package.json file?

Reply:
I considered it, but it will be annoying as we would need to repeat dependencies etc.

A more invasive approach would be to restructure the whole repo:

- src/ng-dev
- src/build-tooling/...

Reply:
Okay, something to probably keep in mind in the future, but lets leave it for now.